app/src/main/java/com/bc/gordiansigner/helper/ext/StringExt.kt

```diff
@@ -1,3 +1,18 @@
 package com.bc.gordiansigner.helper.ext
 
-fun CharSequence.replaceSpaces() = this.replace("\\s+".toRegex(), " ").trim()
\ No newline at end of file
+import com.google.zxing.BarcodeFormat
+import com.google.zxing.EncodeHintType
+import com.google.zxing.MultiFormatWriter
+import io.reactivex.Single
+import io.reactivex.schedulers.Schedulers
+
+fun CharSequence.replaceSpaces() = this.replace("\\s+".toRegex(), " ").trim()
+
+fun String.toQrCode(size: Int, isReverted: Boolean = false) = Single.fromCallable {
```

Comment:
Minor thing. I think it should not be a reactiveX related stuff. It should be just a utility function and if you want to make it as reactiveX stream, please do it in `ViewModel` like `generateQRCode(String)`.